### PR TITLE
WIFI-1669 TIP 1.0 Update image tags in release 1.0 branch

### PR DESCRIPTION
**Pull request by AkshayJagadish-ne**
This should be merged after WIFI-1669 for other components have been merged
https://github.com/Telecominfraproject/wlan-cloud-graphql-gw/pull/20
https://github.com/Telecominfraproject/wlan-cloud-ui/pull/89
https://github.com/Telecominfraproject/wlan-cloud-opensync-protobuf/pull/5
https://github.com/Telecominfraproject/wlan-cloud-workspace/pull/3
https://github.com/Telecominfraproject/wlan-cloud-opensync-controller/pull/30
https://github.com/Telecominfraproject/wlan-cloud-services/pull/80
https://github.com/Telecominfraproject/wlan-cloud-root/pull/3
https://github.com/Telecominfraproject/wlan-cloud-base/pull/10

### CODE REVIEW

**Review by yongchen-cu — APPROVED** [review]
to be merged after SDK frontend/backend and database schema are updated.
versioning tag on master unchanged until TIP decides on the versioning labeling.

File: tip-wlan/charts/opensync-gw-cloud/values.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ replicaCount: 1
 
 image:
   name: opensync-gateway-cloud
-  tag: 0.0.1-SNAPSHOT
+  tag: 1.0.0-SNAPSHOT
```
Comment (dmitry-toptygin-connectus-ai) [inline tip-wlan/charts/opensync-gw-cloud/values.yaml:14]: how about defining a single value release-tag:1.0.0-SNAPSHOT and using it everywhere in the charts ?

Comment (AkshayJagadish-ne) [reply tip-wlan/charts/opensync-gw-cloud/values.yaml:14]: Yes I agree, I will be submitting the changes for that

File: tip-wlan/charts/opensync-gw-static/values.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ replicaCount: 1
 
 image:
   name: opensync-gateway-static
```
Comment (rahulsharma2020) [inline tip-wlan/charts/opensync-gw-static/values.yaml:13]: I think it's about time we remove this component from the charts

**Pull request merged**